lib/iris/tests/unit/analysis/trajectory/test__nearest_neighbour_indices_ndcoords.py

```diff
@@ -105,7 +102,7 @@ def test_no_y_dim(self):
         cube.add_dim_coord(co_x, 1)
```

Comment:
DO NOT ACTION, OUT OF SCOPE.

The first five lines of each test in this class (excluding comments) is duplicated, I think it should be moved into a _setup fixture.

Reply:
https://github.com/SciTools/iris/issues/6625